### ISSUE TITLE
(First time) Request for your tool's exe file

### ISSUE THREAD

**Issue by DJANDREAJAPAN** [mon/infinitas_decrypt#2]
I'm a newcomer to GitHub, but I want to ask to you this question: Would you make an exe file of this infinitas decrypter tool please?
It's because I did tried this tool's python files and did the installation, but there's an error that doesn't compile the 'sys' and the other 'sys' error.
I tried my best to install this python file and let it work, but it still doesn't work. So that is why I'm asking you as an request, to please make a simple exe file like the ifstools please?
the ifstools.exe file is simple and easy to use, because it didn't need any complicated installation.
P.S. I'm also trying to dissect the PC SOUND VOLTEX III as well, but I got stuck because even if I install the camellia egg file, it says "No module named Camellia". If you could make the sv3c decrypter as an exe file as well, I would be satisfied and happy to start my disecting.
Best regards, IIDXROGUE (The Easter egg hunter)

**Comment by mon** [comment]
I saw your hatena blog. I won't make an exe, it's actually quite difficult...

Can you post the exact error? I might be able to help

**Comment by DJANDREAJAPAN** [comment]
Thank you for seeing my hatena blog.
Anyway, I have this struggle at your INFINITAS data extracter:
![image](https://user-images.githubusercontent.com/60337494/73178965-b8ff8300-4155-11ea-92b5-24d230a4c00e.jpg)
And then, it comes out like this:
![image](https://user-images.githubusercontent.com/60337494/73179007-d03e7080-4155-11ea-9c91-b9e969f69d3a.jpg)
It's a shame that even if I install all of the modules for infinitas_decrypt-master, there's this "unsupported operand type(s) for ^: str and str" error that I got stuck at.
Also, for your sv3c decrypter, I got stuck at this section:
![image](https://user-images.githubusercontent.com/60337494/73179205-388d5200-4156-11ea-94d9-28ea80303c12.jpg)
Basically, python is too complicated for me just to install the extractor. That's why I was begging for an exe file of the extractor a while ago.

**Comment by DJANDREAJAPAN** [comment]
Also, I don't know why does it say "No module named camellia", even though if I did install the camellia python file from your tools.
I'm sorry that I didn't know that creating an exe file to ignore the complicated python installation is actually difficult.
